app/controllers/identity.js

```diff
@@ -14,16 +14,49 @@ export default class IdentityController extends Controller {
   @tracked isCopyClicked = false;
   @tracked isVerifyClicked = false;
   @tracked profileURL = this.model.profileURL || '';
+  @tracked saveDisabled = true;
+  @tracked generateChainCodeDisabled = this.model.profileURL === undefined;
+  @tracked checkboxDisabled =
+    this.generateChainCodeDisabled || this.model.chaincode === undefined;
+  @tracked linkDisabled = true;
 
   @action async handleRefresh() {
     window.location.reload();
   }
 
+  @action changeSaveDisabled() {
+    const isValidUrl = (urlString) => {
+      try {
+        return Boolean(new URL(urlString));
+      } catch (e) {
+        return false;
+      }
+    };
+    if (
+      this.profileURL === '' ||
+      this.profileURL === (this.model.profileURL || '') ||
+      !isValidUrl(this.profileURL)
+    ) {
+      this.saveDisabled = true;
+    } else {
+      this.saveDisabled = false;
+    }
+  }
+
+  @action changeLinkDisable() {
+    if (!this.isChecked) {
+      this.linkDisabled = false;
+    } else {
+      this.linkDisabled = true;
+    }
```

Comment:
Do we actually need this action??
Can we not directly use the `isChecked` variable with a ! (not) operator wherever needed

Reply:
I tried doing this, it changes `this.isChecked` into a comment, and I tried to put that in another variable and then use that as a value, but that also didn't work

Reply:
You are doing this in `.hbs` file right ??
It's different in there I guess